packages/SketchMorph2-Tests.package/M2DrawingComponentTests.class/instance/drawAmountOfDiagonalCircles..st

```diff
@@ -6,4 +6,4 @@ drawAmountOfDiagonalCircles: aNumber
 			userActionMoveDrawing: ((4 * x) + 2 ) asPoint;
```

Comment:
I don't think there should be a space in front of the closing parenthesis.

Reply:
Good point. I'll try to add that.